pages/api/createuser.ts

```diff
@@ -41,12 +41,16 @@ export default async function handler(
         phone: phone_number,
         banned: banned,
         createdAt: created_at.toString(),
-        updatedAt: updated_at.toString()
+        updatedAt: updated_at.toString(),
+        // always make default users be patients
+        role: 'patient'
```

Comment:
please use the enum/type to declare this as a patient

Reply:
done